docs/self-managed/zeebe-deployment/operations/management-api.md

```diff
@@ -35,6 +35,12 @@ POST actuator/exporting/pause
 
 When all partitions pause exporting, a successful response is received. If the request fails, some partitions may have paused exporting. Therefore, it is important to either retry until success or revert the partial pause by resuming exporting.
 
+Another option is to use the soft pause feature, which enables us to continue to export records without updating the last acknowledged record position. This allows us to continue to export data to ES without creating new snapshots while we do the backups.
```

Comment:
I would suggest to add a separate section for soft pause. 

For the explanation, I would write something along the lines of the following. No need to mention last acknowledged position: 

Soft pause feature can be used when you want to continue exporting records but do not want to delete those records (log compaction) from zeebe. This is particularly useful during hot backups. How to use this feature for hot backups is documented [here](link to hot backup docs).